packages/builder/src/steps/provision.test.ts

```diff
@@ -84,7 +84,7 @@ describe('steps/provision.ts', () => {
       const result = await action.getState(
         fakeRuntime,
         fakeCtx,
-        { source: 'hello:1.0.0', sourcePreset: 'foobar', chainId: 1234, targetPreset: 'voop', options: { bar: 'baz' } },
```

Comment:
tests were previously testing with foobar rather than main likely for a reason... why did it become necessary to change to main?

Reply:
The package references in the tests are using main as the preset and the tests were failing because it was prioritizing the sourcePreset.